Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1825,6 +1828,52 @@ namespace PlayerCommands
 		}
 	}
 
+	void GetNecessitiesStatus(uint client, const wstring &args) {
+		// Check that this player is in a player controlled base
+		PlayerBase *base = GetPlayerBaseForClient(client);
+		if (!base)
+		{
+			PrintUserCmdText(client, L"ERR Not in player base");
+			return;
+		}
+
+		if (!clients[client].admin && !clients[client].viewshop)
+		{
+			PrintUserCmdText(client, L"ERROR: Access denied");

```

Comment:
This should probably actually be `ERR Access denied` for consistency with most of the rest of the plugin, instead of /shop.

Reply:
Submitted.